README.md

```diff
@@ -248,6 +248,15 @@ Build and install aws-c-common:
 
 You have successfully built and installed the AWS Encryption SDK for C.
 
+## Doxygen Documentation
+
+To build the documentation, you'll need [doxygen](http://www.doxygen.nl/) installed.  Check the [downloads](http://www.doxygen.nl/download.html) page 
+or use your OS package manager.
+
+Next, rerun the above cmake with a `-DBUILD_DOC="ON"` flag before building aws-encryption-sdk-c.
+
+Finally, run either `make doc_doxygen` (Unix) or `xcodebuild -scheme doc_doxygen` (Mac) to generate the documentation.
```

Comment:
Windows instructions for Doxygen is missing here.

Reply:
addressed in next commit